### ISSUE TITLE
feature request: invert bg and fg colors in selection

### ISSUE THREAD

**Issue by musou1500** [zellij-org/zellij#2395]
It may be useful to invert the colors with the following settings.

```kdl
ui {
    selection {
        invert_colors true
    }
}
```

In current behavior, zellij uses `bg` color as selection background color.
if terminal emulator has similar color scheme to zellij, this behavior reduces visibility.
I tried invert bg and fg colors in selection, visibility looks better.

![image](https://user-images.githubusercontent.com/5037195/233796786-cbcc379c-069a-4d32-8738-fa4763b8ecfa.png)

**Comment by d3r1n** [comment]
Thanks for the solution. I use gruvbox as my alacritty and zellij theme. Selection color wasn't even showing when I switched to zellij.

**Comment by jgayfer** [comment]
Definitely needed! The default gruvbox dark theme isn't really usable if you are also using gruvbox in your terminal (which I imagine most people are).

**Comment by truehumandesign** [comment]
I also need this very badly - I hope #2488 gets some more attention

**Comment by sascha-wi** [comment]
Yes please, I can't see the selection because I use similar colors for zellij and shell/terminal.

**Comment by davidhooey** [comment]
I too have the same issue using everforest-dark in Alicritty and Zellij. Selection is very faint and hard to see.
